components/progress-bar/src/ProgressBar.js

```diff
@@ -5,90 +5,91 @@ import PropTypes from 'prop-types';
 import { jsx, useTheme } from '@westpac/core';
 import { SrOnly } from '@westpac/accessibility-helpers';
 
+// ==============================
+// Utils
+// ==============================
+
+const round = value => Math.round(value);
+
 // ==============================
 // Component
 // ==============================
 
 /**
- * Progress Bar: A visual indication of progress. Use when loading content or to indicate how far along the user is in a journey.
+ * Progress Bar: A visual indication of progress.
+ * Use when loading content or to indicate how far along
+ * the user is in a journey.
  */
-export const ProgressBar = ({ value, isSkinny, ...props }) => {
-	const { progressBar } = useTheme();
-
-	const valueRound = Math.round(value);
-
-	const size = isSkinny ? 'skinny' : 'default';
-
-	const style = {
-		// Common styling
-		common: {
-			height: progressBar.size[size].height,
-			marginBottom: progressBar.marginBottom,
-			overflow: 'hidden',
-			backgroundColor: 'white',
-			borderRadius: progressBar.size[size].height,
-			border: `${progressBar.borderWidth} solid ${progressBar.borderColor}`,
-			padding: progressBar.padding,
-			position: 'relative',
+export const ProgressBar = ({ value, skinny, ...props }) => {
+	const { COLORS } = useTheme();
 
-			'::after': {
-				display: isSkinny && 'none',
-				content: '"0%"',
-				position: 'absolute',
-				left: '1rem',
-				top: 0,
-				height: '100%',
-				color: '#575f65',
-				fontSize: '1.4rem',
-				fontWeight: 700,
-				zIndex: 1,
-			},
-		},
+	const roundedValue = round(value);
```

Comment:
I computed the rounded value once here, and use it in 2 place instead of invoking the `round()` function twice with the same argument.